test/common/testlib.py

```diff
@@ -1705,7 +1705,6 @@ def login_and_go(self, path: Optional[str] = None, user: Optional[str] = None, h
         # HACK: Fix these ASAP, these are major bugs
         "Warning: validateDOMNesting.*cannot appear as a",
         "Warning: React does not recognize the.*prop on a DOM element",
-        "Warning: .*prop on .* should not be null",
```

Comment:
This was in a PR I just merged no?

Reply:
Yes, but I included it here as it would conflict. Sorry, this one should have had draft/blocked.